lib/xpm_ruby/connection.rb

```diff
@@ -3,11 +3,12 @@
 
 module XpmRuby
   class Connection
-    attr_accessor :xero_tenant_id, :authorization
+    attr_accessor :xero_tenant_id, :authorization, :url
 
-    def initialize(access_token:, xero_tenant_id:)
+    def initialize(access_token:, xero_tenant_id:, url: nil)
```

Comment:
Alternative style part 1:
```suggestion
    def initialize(access_token:, xero_tenant_id:, url: xpm_url)
```

Reply:
Hey Dave yeah I'd prefer using the param default usually but here the upstream methods would then need to call .new with no url param or with an overridden url param (or check for nil) with a branch rather than passing on a url param from even further upstream.. if that makes sense.

Example here:
https://github.com/ignitionapp/Practice-Ignition/blob/288ef4573c7aad3bfb3ce007be20b1a04a444476/subdomains/xpm_integration/staff.rb#L29-L38

Reply:
Oh wow, I didn't know Ruby keyword args behaved like that.  Thanks for explaining 👌